include/hyro/Thresholding.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#ifndef SIGNAL_GENERATOR_H
+#define SIGNAL_GENERATOR_H
+
+#include <bits/stdc++.h>
+
+namespace hyro
+{
+
+class Thresholding 
+{
+  private:
+
+    double amplitude, threshold;
```

Comment:
Usually the member of class are naming with a suffix `m_`

In this case, `m_amplitude` and `m_threshold`